src/Performance/Minibench.purs

```diff
@@ -12,17 +12,16 @@ module Performance.Minibench
   , withUnits
   ) where
 
-import Prelude hiding (min,max)
+import Prelude hiding (min, max)
 
 import Data.Int (toNumber)
+import Data.Number (infinity, max, min, sqrt)
 import Effect (Effect, forE)
 import Effect.Console (log)
 import Effect.Ref as Ref
-import Effect.Uncurried (EffectFn1, runEffectFn1)
-import Data.Number (infinity, max, min, sqrt)
 
 -- | Returns the number of nanoseconds it takes to evaluate the given closure.
-foreign import timeNs :: forall a. EffectFn1 (Unit -> a) Number
+foreign import timeNs :: forall a. (Unit -> a) -> Effect Number
```

Comment:
Whenever I write FFI I try to avoid uncurried parameters. Could have written another way, but im not really sure how this matters if the API remains the same.

Reply:
The `EffectFn1` is more "uncurried" because you can omit the inner thunk in the foreign definition. See the JS definition [here](https://github.com/purescm/purescript-minibench/blob/master/src/Performance/Minibench.js#L1).

Anyways it would be good to keep this the same as for the JS package, just so that we deviate as little as possible and it's easier to merge in changes from upstream in the future. Also changing this would mean that the JS version is incorrect which still exists in this branch.